DotNet/DataAcquisition/Controllers/ConnectionValidationController.cs

```diff
@@ -32,11 +33,14 @@ public ConnectionValidationController(ILogger<ConnectionValidationController> lo
         string facilityId, 
         [FromQuery] string? patientId = default, 
         [FromQuery] string? patientIdentifier = default,
+        [FromQuery] string? measureId = default,
```

Comment:
Add new parameters to code-docs above (`<param>`)

Reply:
done and then some